pandas/_libs/window.pyx

```diff
@@ -30,19 +35,9 @@ cdef inline int int_min(int a, int b): return a if a <= b else b
 
 from util cimport numeric
 
-from skiplist cimport (
-    skiplist_t,
-    skiplist_init,
-    skiplist_destroy,
-    skiplist_get,
-    skiplist_insert,
-    skiplist_remove)
-
-cdef extern from "../src/headers/math.h":
```

Comment:
i wonder if this makes any difference (the sqrt & log above)

Reply:
It'd be pretty weird (isn't the path there incorrect anyway?) but no harm in reverting it just to see...

Since skiplist is only used in window, and used to be "include"d anyway, I think pasting it at the bottom of window is a decent fallback option.

Reply:
Pending double-checking... reverting to getting sqrt from src/headers/math.h seems to make up the difference and then some.  It looks like the sqrt exposed in libc.math is not declared with `nogil`; that could plausibly make a difference.

Reply:
OK, reverting the `sqrt` import _does_ appear to have closed the gap.

```
taskset 2 asv continuous -f 1.1 -E virtualenv master HEAD -b rolling -b quantile -b median
[...]
BENCHMARKS NOT SIGNIFICANTLY CHANGED.

taskset 2 asv continuous -f 1.1 -E virtualenv master HEAD -b rolling -b quantile -b median
[...]
       before           after         ratio
     [2dbf2a6a]       [4432b71c]
-           2.61s            2.35s     0.90  rolling.DataframeRolling.time_rolling_cov
-           2.71s            2.23s     0.82  rolling.DataframeRolling.time_rolling_corr

taskset 2 asv continuous -f 1.1 -E virtualenv master HEAD -b rolling -b quantile -b median
[...]
BENCHMARKS NOT SIGNIFICANTLY CHANGED.
```

I'll push with this change and open an issue to check for other places where libc.sqrt is cimported instead of the nogil version.